doc/versioning.md

```diff
@@ -0,0 +1,59 @@
+# Plan for versioning support
+
+### Record-based
+The plan is to use the records format from https://github.com/equinor/records and https://github.com/equinor/revisions for the P&IDs
+For the first demo, I suggest just using plain records with no replaces and no describes, with the translation of each
+dexpi file in a separate record.
+
+### Single-command startup
+The demo should be possible to start with a single command-line. The best option is probably to keep a local directory with all versions of a P&ID and then use docker's volume mapping to give the docker containers access to it
+Each file will be translated with RML-mapper and then put into its own record. The record will have an IRI (or some property ex:fileName) based on the filename so the frontend can communicate with the backend about each file.
+
+
+### Graphical info in RDF
+The setup will be a bit different with graphical info in RDF, mainly the actions in the front-end when selecting a new version
+In the current setup, this will be based on file-loading, while later only interaction with the triplestore is needed.
+
+### Separate storage of commissioning packages
+The commissioning packages (including internal element(s), boundary points and perhaps color, but not including the scoping) must be stored separately from the P&IDs
+For this first demo, storage in the default graph should be sufficient.
+
+### Identifiers
+This setup assumes that the identifiers we use for the boundaries do not change across versions. Ids that change across versions must not be used for elements of the commissioning package. (But they can be highlighted as part of scope)
```

Comment:
Mostly agree! I think that they can be part of the commissioning package, because the tag information associated with those elements might be of relevance? I think it more correct that these Ids cannot be selected as boundaries or internal components since they are not persistent. Selecting these IDs would not make it possible to apply the same boundary on a different version of that P&ID where these IDs would not be present.